modules/installation-nutanix-installer-infra-reqs.adoc

```diff
@@ -11,7 +11,62 @@ Before you install an {product-title} cluster, review the following Nutanix AOS
 [id="installation-nutanix-installer-infra-reqs-account_{context}"]
 == Required account privileges
 
-Installing a cluster to Nutanix requires an account with administrative privileges to read and create the required resources.
+The installation program requires access to a Nutanix account with the necessary permissions to deploy the cluster, as well as to maintain the daily operation of it. The following options are available to you:
+
+* You can use a local Prism Central user account with administrative privileges. Using a local account is the simplest way to grant access to an account with the required permissions.
+* If your organization’s security policies require that you use a more restrictive set of permissions, use the permissions in the following table to create a custom Cloud Native role in Prism Central. You can then assign the role to a user account that is a member of a Prism Central authentication directory. When assigning entities to the role, ensure that the user can only access the Prism Element and subnet that are required to deploy the virtual machines.
```

Comment:
> ensure that the user can only access the Prism Element
Should this be _ensure that the user can access only the Prism Element_, which means the user can access the Prism element and nothing else?

Reply:
Thanks for catching this @mburke5678. Updated.